transform/mattermost-analytics/models/intermediate/product/licenses/_licenses_servers/int_self_hosted_servers.sql

```diff
@@ -0,0 +1,21 @@
+with mm_telemetry_prod_license as (
+    select timestamp::date as license_date
+        , license_id as license_id
+        , server_id as server_id
+        , customer_id as customer_id
+        , license_name as license_name
```

Comment:
A server might have multiple license ids (i.e. renewals). The `qualify` statement will keep only the latest license id.

Since this table also handles historical feedback, does it make sense to use the latest license? An alternative would be the license at the date of the event.

Reply:
I added `license_id` to the qualify, so we get all licenses for all servers